src/app/vault/ciphers.component.html

```diff
@@ -7,6 +7,70 @@
     [infiniteScrollDisabled]="!isPaging()"
     (scrolled)="loadMore()"
   >
+    <thead id="sort-ciphers-table">
+      <tr>
+        <th scope="col">
+          <input (click)="selectAll()" type="checkbox" />
+        </th>
+        <th
+          colspan="2"
+          *ngIf="sortedDescending; else elseBlock1"
+          scope="col"
+          class="show-arrow-on-hover user-select-none"
+        >
+          <i
+            (click)="sortCiphers('name')"
+            class="sorting-arrow sort-name-arrow bwi-arrow-down"
+            aria-hidden="true"
+          ></i
+          >{{ "name" | i18n }}
+        </th>
+        <ng-template #elseBlock1>
+          <th colspan="2" scope="col" class="show-arrow-on-hover user-select-none">
+            <i
+              (click)="sortCiphers('name')"
+              class="sorting-arrow sort-name-arrow bwi-arrow-up"
+              aria-hidden="true"
+            ></i
+            >{{ "name" | i18n }}
+          </th>
+        </ng-template>
+        <th
+          scope="col"
+          colspan="2"
+          class="show-arrow-on-hover table-col-sort cipher-sort-table-header"
+        >
+          <div class="input-group m-0 font-weight-bold">
+            <div
+              (click)="sortCiphers(option.value)"
```

Comment:
Instead of referring to `option.value`, refer to the class property that control is bound to. e.g. `showData` (or whatever you rename it to)